src/System.Private.CoreLib/ILLinkTrim.xml

```diff
@@ -72,5 +72,9 @@
     <!-- Workaround for https://github.com/mono/linker/issues/378 -->
     <type fullname="System.Runtime.InteropServices.IDispatch" />
     <type fullname="Internal.Runtime.InteropServices.IClassFactory2" />
+    <type fullname="System.GC">
+      <!-- Internal API that we are iterating on: https://github.com/dotnet/coreclr/pull/24202 -->
+      <method name="RegisterMemoryLoadChangeNotification" />
```

Comment:
You should add `UnregisterMemoryLoadChangeNotification` too.

Reply:
(Or drop this from this change completely. This API was broken for quite a while and nobody noticed.)